packages/wallets/src/features/accounts/modules/Poa/Poa.tsx

```diff
@@ -41,6 +44,8 @@ const Poa: React.FC<TPoaProps> = ({ onCompletion }) => {
         }
     };
 
+    if (showLoader) return <Loader />;
```

Comment:
why not directly use `isLoading` instead of creating another state?

Reply:
It was like that before. But, for some reason it was giving memory leak error. So, i had to create another state to avoid that error.